### ISSUE TITLE
The OpenWeather extension doesn't show up so I can't move it

### ISSUE THREAD

**Issue by FastThenLeft** [andia89/order-icons#7]
Ubuntu 22.04. The OpenWeather extension shows up in the center. But, when I try and change the order of the center with your extension, OpenWeather doesn't show up so I cannot move it. Thanks for looking into this!

**Comment by andia89** [comment]
Yeah I can confirm it doesn't work. No idea why though. I will try and investigate

**Comment by FastThenLeft** [comment]
Thanks!

**Comment by InklingGirl** [comment]
Any update @andia89?
